utils/src/test/kotlin/UtilsTest.kt

```diff
@@ -428,6 +428,70 @@ class UtilsTest : WordSpec({
         }
     }
 
+    "redirecting output" should {
+        "work for stdout only" {
+            val stdout = redirectStdout {
+                for (i in 1..10000) System.out.println("stdout: $i")
```

Comment:
Please explain in a code comment why it is necessary to test 10000 lines, and why just a few lines are not enough.